syntax = "proto3";
// ??message??????
message rsp_data {
	int64 user_id=1; //????
	string token=2;

}
message article {

}
message req_data {
	bool user_info=1;
	repeated article article=2;
}

// ??service??????
service name {
//??rpc????????
rpc tokrn_interface(rsp_data) returns(req_data){};

}